### PR TITLE
Security Scanners information in Module Template

### PR DESCRIPTION
**Pull request by khlifi411**
**Description**

Changes proposed in this pull request:

- Introduce new step in create module command to add security scanning data from `sec-scanners-config.yaml` file provided by module developer teams

### CODE REVIEW

**Review by khlifi411 — COMMENTED** [review]
**To reviewer**
The current implementation will fail when a `sec-scanners-config.yaml` is not provided. As this requirement wasn't communicated to team developers yet, should we fail silently when the file is not found?

File: cmd/kyma/alpha/create/module/module.go
Comment (jakobmoellersap) [inline cmd/kyma/alpha/create/module/module.go:162]: can we change this so that the step is only activated if the file is found? that way you can silently opt out of this process if the file is not present. Alternative would be to define a bool flag which is disabled by default and can take in a file

File: pkg/module/security_scan.go
Comment (khlifi411) [reply pkg/module/security_scan.go:24]: The lifecycle manager complains that the `ociRegistry` access type is not supported. I will raise a PR to introduce support for it 👍

**Review by jakobmoellersap — COMMENTED** [review]
LGTM, needs a second review and then we are good to go I believe 👍

File: pkg/module/security_scan.go
Comment (ruanxin) [inline pkg/module/security_scan.go:88]: better mentioned in the [original issue](https://github.tools.sap/kyma/backlog/issues/3338) that the access layer format is different than proposed (there is no ref field), and also highlight it's according to gardener component-spec api definition https://github.com/gardener/component-spec/blob/master/bindings-go/apis/v2/accesstypes.go

```
  - access:
      imageReference: artifact.registry.hostname/path/to/the/registry/k8s-tools:v20221213-1840f45a
      type: ociRegistry
    labels:
    - name: type.scan.security.kyma-project.io
      value:
        type: third-party-image
    name: k8s-tools
    relation: external
    type: ociImage
    version: 0.3.4
```

**Review by ruanxin — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**
